### PR TITLE
A-D review: Various fixes in the Security Considerations section

### PR DESCRIPTION
**Pull request by robstradling**

### CODE REVIEW

File: draft-ietf-trans-rfc6962-bis.md [hunk 1]
```diff
@@ -2048,7 +2046,7 @@ Misissued certificates that have not been publicly logged, and thus do not have
 a valid SCT, are not considered compliant. Misissued certificates that do have
 an SCT from a log will appear in that public log within the Maximum Merge Delay,
 assuming the log is operating correctly. As a log is allowed to serve an STH
-that's up to MMD old, the maximum period of time during which a misissued
+that's up to the MMD old, the maximum period of time during which a misissued
```
Comment (eranmes) [inline draft-ietf-trans-rfc6962-bis.md]: Not sure the 'the' appropriate here?
My original intention was to say that the period an old STH may be served is up to this STH's age + MMD. Have not expressed it well, I believe.

File: draft-ietf-trans-rfc6962-bis.md [hunk 2]
```diff
@@ -2069,10 +2067,8 @@ provides more details on how this can be done.
 
 Violation of the MMD contract is detected by log clients requesting a Merkle
 inclusion proof ({{get-proof-by-hash}}) for each observed SCT. These checks can
-be asynchronous and need only be done once per certificate. In order to protect
-the clients' privacy, these checks need not reveal the exact certificate to the
-log. Instead, clients can request the proof from a trusted auditor (since anyone
-can compute the proofs from the log) or communicate with the log via proxies.
+be asynchronous and need only be done once per certificate. However, note that
+there may be privacy concerns (see {{fetching_inclusion_proofs}}).
```
Comment (eranmes) [inline draft-ietf-trans-rfc6962-bis.md]: This references a section that was probably added in another PR?

Comment (eranmes) [reply draft-ietf-trans-rfc6962-bis.md]: Should also make sure that the referred section does indeed mention the privacy issues implied in the removed text.

File: draft-ietf-trans-rfc6962-bis.md (hunk 1 above)
Comment (robstradling) [reply draft-ietf-trans-rfc6962-bis.md]: EKR wrote "Nit: up to the MMD old".

I've had another go at expressing it better.

File: draft-ietf-trans-rfc6962-bis.md (hunk 2 above)
Comment (robstradling) [reply draft-ietf-trans-rfc6962-bis.md]: Yes, this section reference was added in another of my PRs, so I didn't duplicate that addition here.

Comment (robstradling) [reply draft-ietf-trans-rfc6962-bis.md]: The referred section does indeed mention the privacy issues.

**Pull request merged**